src/Models/NodesInterface.php

```
@@ -0,0 +1,39 @@
+<?php
+
+declare(strict_types=1);
+
+namespace OpenFGA\Models;
+
+/**
+ * @psalm-type NodesShape = list<NodeShape>
+ */
+interface NodesInterface extends IndexedCollectionInterface
```

PHPStan (2.1.15)

Interface OpenFGA\Models\NodesInterface extends generic interface OpenFGA\Models\IndexedCollectionInterface but does not specify its types: T


(missingType.generics)